src/app/scheduling/daterangepicker/daterangepicker-basic/daterangepicker-basic.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from "@angular/core";
+import { DateRange } from "igniteui-angular";
+
+@Component({
+    selector: "basic-rangedatepicker",
+    styleUrls: ["./daterangepicker-basic.scss"],
+    templateUrl: "./daterangepicker-basic.html"
+})
+export class BasicDateRangePickerComponent {
+    public range: DateRange = { start: new Date(), end: new Date(new Date().setDate(new Date().getDate() + 5)) };
```

Comment:
Make date initialization simpler.
```suggestion
    public range: DateRange = { start: new Date('2000, 4, 20'), end: new Date('2000, 4, 25') };
```

Reply:
For code snippet - yes. But for demo it's better to show current dates.